site/content/blog/release-0.30.0.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Release 0.30.0 is out!"
+date: 2023-02-24T09:50:29+11:00
```

Comment:
I guess the date should be set to when we release?

Reply:
Good point, used a TODO placeholder - this makes `hugo` fail to build the site so we cannot forget.

Reply:
To be explicit, PR now includes a date and i'm updating it ever day so this is mergable (and deployable) without me being available (timezones).